src/Util.h

```diff
@@ -116,4 +117,9 @@ struct PointXy {
     }
 };
 
+// Map for enmu CARTA:FileType to string
+static std::unordered_map<CARTA::FileType, string> ExportTypeToString{{CARTA::FileType::CASA, "CASA"}, {CARTA::FileType::CRTF, "CRTF"},
```

Comment:
Please rename this to something like `FileTypeToString` or maybe `FileTypeString` -- it's being used in export-related code, but it should be named more generically.

Reply:
Rename it